tests/emp_search.py

```diff
@@ -45,6 +49,33 @@ def test_search_by_job_title(self):
 
         for item in list_of_web_elements_jtitles:
             self.assertEqual('SDET', item.text)
+    def test_search_by_employee_id(self):
+        self.login('admin', 'password')
+        time.sleep(1)
+    #     wait = WebDriverWait(self.browser, 10)
+    #     element = wait.until(EC.visibility_of_element_located((By.ID, 'empsearch_id')))
+        self.browser.find_element_by_id('empsearch_id').send_keys('635919')
+        self.browser.find_element_by_id('searchBtn').click()
+        actual_search_result =  self.browser.find_element_by_xpath('//*[@id="resultTable"]/tbody/tr/td[2]').text
+        print('result' +  actual_search_result)
+        self.assertTrue('635919' == actual_search_result)
+        list_of_web_elements_jtitles = self.browser.find_elements_by_xpath('//*[@id="resultTable"]/tbody/tr/td[5]')
```

Comment:
Here you are finding all the job titles, however in this instance searching for rows would be sufficient.
You can remove the TD from the Xpath